ethereum/package.json

```diff
@@ -40,6 +41,7 @@
     "hardhat-gas-reporter": "^1.0.4",
     "ts-node": "^10.0.0",
     "typescript": "^4.3.2",
-    "web3": "^1.3.6"
+    "web3": "^1.3.6",
+    "@snowfork/snowbridge-types": "link:../types"
```

Comment:
Our types aren't necessary in this context.

Reply:
fixed 5186f86